Core/clim-core/presentations.lisp

```diff
@@ -501,6 +500,20 @@ filled in."
 (defmethod get-ptype-metaclass ((type class))
   type)
 
+;;; The presentation type specifier may be in one of these forms:
+;;;
+;;;     name
```

Comment:
Maybe move this comment to the generic function since the "name" case is handled by a different method?

Reply:
fixed.